tests/wpt/tests/fetch/range/cache-revalidate-construct-range-response.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+function cache_revalidation_test(expect, label) {
+    promise_test(async t => {
+        const url = new URL('resources/video-with-aged-proxy-cache.py', location.href);
+        const response_1 = await fetch(url.toString());
+        assert_equals(response_1.status, 200, 'Initial fetch should succeed with 200');
+        const response_1_blob = await response_1.blob();
+        assert_equals(response_1_blob.size, 80666, 'Initial fetch should get full content');
+        const response_2 = await fetch(url.toString(), {
+            headers: {'Range': 'bytes=0-99'}
+        });
+        assert_equals(response_2.status, 206, 'Range fetch after cache revalidation should succeed with 206');
```

Comment:
Firefox and Safari appear to return 200 instead: https://wpt.fyi/results/fetch/range/cache-revalidate-construct-range-response.html?pr=55613 This test should pass on those browsers as well. Can we update the test so that it correctly runs there and then update Servo to return the same results?